capa/features/extractors/viv/insn.py

```diff
@@ -271,7 +271,7 @@ def extract_insn_bytes_features(fh: FunctionHandle, bb, ih: InsnHandle) -> Itera
             if capa.features.extractors.helpers.all_zeros(buf):
                 continue
 
-            if f.vw.isProbablyString(v):
+            if f.vw.isProbablyString(v) or f.vw.detectUnicode(v):
```

Comment:
```suggestion
            if f.vw.isProbablyString(v) or f.vw.isProbablyUnicode(v):
```

https://github.com/vivisect/vivisect/blob/91e8419a861f49779f18316f155311967e696836/vivisect/__init__.py#L1113